lib/Stats.tsx

```diff
@@ -1,101 +1,236 @@
-import React, { useMemo } from 'react'
+import React, { useCallback, useEffect, useRef, useState } from 'react'
 import styled from 'styled-components'
 import { VapixParameters, VideoProperties } from './PlaybackArea'
 
+import { StreamStats } from './img'
+
 const StatsWrapper = styled.div`
-  background: rgb(0, 0, 0, 0.66);
   position: absolute;
   top: 24px;
   left: 24px;
-  padding: 8px 16px;
-  font-family: sans-serif;
+  width: 360px;
+  min-width: 240px;
   max-width: 80%;
   max-height: 80%;
+  border-radius: 4px;
+  background: #292929 0% 0% no-repeat padding-box;
+  opacity: 0.88;
+  overflow: scroll;
```

Comment:
Remove `overflow: scroll`. Not only is it giving me scrollbars that seem to be quite uneeded. It also seems to mess up the sharpness of the text for some reason.

Compare:

![scroll_on](https://user-images.githubusercontent.com/7765599/98987585-ac9eb800-2526-11eb-83ea-c0a66cdc7d10.png)
![scroll_off](https://user-images.githubusercontent.com/7765599/98987587-ad374e80-2526-11eb-878b-63aaa0d9389d.png)

Reply:
It seems to be overflowing because the grid is larger than the container. The grid is almost 400px wide with all the gaps and paddings but the container is 360px.

Reply:
Done